packages/core/src/codewhisperer/util/authUtil.ts

```diff
@@ -151,6 +151,7 @@ export class AuthUtil {
         }
 
         await setContext('aws.codewhisperer.connected', this.isConnected())
+        await setContext('aws.isInternalUser', this.isInternalAmazonUser())
```

Comment:
is this something that should live in the general activation since it's not necessarily relevant to codewhisperer?

Reply:
I think this makes sense.